{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -66,11 +66,14 @@ env:
         - PYTHON_VERSION=2.7 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.6 SETUP_CMD='egg_info'
+        - PYTHON_VERSION=3.7 SETUP_CMD='egg_info'
         {%- elif  cookiecutter.minimum_python_version == "3.5" %}
         - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.6 SETUP_CMD='egg_info'
+        - PYTHON_VERSION=3.7 SETUP_CMD='egg_info'
```

Comment:
For egg_info builds I don't think we need more than one Python 3 version, so can you remove 3.5 and 3.6 in the two cases above?

Reply:
This is a good point, currently in most packages we build the egg_info for all supported versions. Should we then drop all but one for python2 and one for python3?